BeeTracker/app/src/main/res/values/strings.xml

```diff
@@ -2,4 +2,6 @@
     <string name="app_name">Bee Tracker</string>
     <string name="facebook_app_id">221233091726222</string>
     <string name="fb_login_protocol_scheme">fb221233091726222</string>
+    <string name="title_activity_submission">SubmissionActivity</string>
```

Comment:
Not important for this pull request but in the future you should put all new strings in this file. So the button titles should go here, so should the hello world text.